src/Security/src/Authentication.JwtBearer/PostConfigureJwtBearerOptions.cs

```diff
@@ -42,7 +43,18 @@ public void PostConfigure(string? name, JwtBearerOptions options)
             return;
         }
 
-        options.TokenValidationParameters.ValidIssuer = $"{options.Authority}/oauth/token";
+        if (Platform.IsCloudFoundry && options.Authority.Contains(".login", StringComparison.OrdinalIgnoreCase))
+        {
+            options.TokenValidationParameters.ValidIssuers =
+            [
+                $"{options.Authority}/oauth/token",
+                $"{options.Authority.Replace(".login", ".uaa", StringComparison.OrdinalIgnoreCase)}/oauth/token"
```

Comment:
Why case-insensitive?

Reply:
It's evaluating a hostname, which wouldn't be case sensitive. I suspect it would always be lowercase, so maybe it doesn't need to be, but I don't know that for sure and figured this wouldn't hurt